src/glossary.rst

```diff
@@ -1767,3 +1767,17 @@ Glossary
       :term:`incomplete outputs <output completion>`, then the workflow is
       not complete and the scheduler will :term:`stall` pending manual intervention.
 
+   dummy task
+      A task which runs a trivially simple script such as ``sleep 1``,
+      ``exit 0`` or ``true``, or which uses alternative :ref:`task-run-modes.skip`
+      or :ref:`workflow-run-modes` to run such a script or avoid running a
+      script at all.
+
+      .. important::
+
+         All tasks in a workflow run in dummy or simulation mode are dummy
```

Comment:
```suggestion
         All tasks that run in dummy or simulation mode are dummy
```

(At first glance I saw "All tasks in a workflow run in dummy or simulation mode" 🤣  )

Reply:
No, it's deliberate? Am I missing something?